lib/service_check.pm

```diff
@@ -181,6 +181,11 @@ service package name.
 
 sub _is_applicable {
     my ($srv_pkg_name) = @_;
+    if ($srv_pkg_name eq 'kdump' && check_var('ARCH', 's390x')) {
+        # workaround for bsc#116300 on s390x
+        record_soft_failure 'bsc#1163000';
```

Comment:
It's better to give bug description in the record_soft_failure, not only the bug id.

Reply:
ok, changed.